DependencyInjection/Compiler/LoaderCompilerPass.php

```diff
@@ -27,20 +27,9 @@ public function process(ContainerBuilder $container)
     {
         $loaders = [];
         foreach ($container->findTaggedServiceIds('coop_tilleuls_migration.loader') as $id => $attributes) {
-            if (!isset($attributes[0]['alias']) || empty($attributes[0]['alias'])) {
-                throw new \LogicException(sprintf(
-                    'You must provide an `alias` attribute to the tag "coop_tilleuls_migration.loader" for service "%s".',
-                    $id
-                ));
-            }
-            $priority = isset($attributes[0]['priority']) ? (int) $attributes[0]['priority'] : 0;
-            $loaders[$priority][$attributes[0]['alias']] = new Reference($id);
+            $loaders[] = new Reference($id);
```

Comment:
If you bump `symfony/dependency-injection`to `3.3`, this is a good example to use a service locator! Same for your transformers! 😉